Adapt the dropzone on the design of the product page

Quetzacoalt91 opened the pull request:
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Look at my dropzone. My dropzone is beautiful.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | [BOOM-638](http://forge.prestashop.com/browse/BOOM-638) & [BOOM-847](http://forge.prestashop.com/browse/BOOM-847)
| How to test?  | Everything to review is in the Jira issue. <ul><li>Check the icon fits the screenshot</li><li>Only the + mark can now open the file manager</li><li>The image form has changed (with animations, wow)</li><li>The image previews are now smooth</li><li>No dashed border all around empty dropzone</li><li>Progress bar upload & image product form now fit the mock up</li>
</ul>

Quetzacoalt91 commented:
![capture du 2016-07-01 17-06-47](https://cloud.githubusercontent.com/assets/6768917/16525773/3c19633a-3fae-11e6-9d4d-33e25cd15bdb.png)

maximebiloe commented:
![ozone](https://cloud.githubusercontent.com/assets/2203436/16525931/0f4e5ae4-3faf-11e6-92e9-539ab2626272.jpg)

Dropzone? 🎶  Ma-ia-hii, Ma-ia-huu 🎶

fojtcz commented on admin-dev/themes/default/js/bundle/product/form.js:
```diff
@@ -1233,19 +1245,20 @@ var imagesProduct = (function() {
         addRemoveLinks: true,
-        clickable: true,
-        thumbnailWidth: 130,
+        clickable: ".openfilemanager",
```
single quote 😄

vincentbz commented:
Got one !
If I delete the images, I have the two borders, there should be only the dotted one

![image](https://cloud.githubusercontent.com/assets/7567060/16617679/51264b1e-4385-11e6-9853-f508e6ed226d.png)

Quetzacoalt91 commented:
PR updated:
* Removed dashed border all around empty dropzone
* Updated image product form (action buttons)
* Fixed [BOOM-847](http://forge.prestashop.com/browse/BOOM-847)
* Set proper style on progress bar

dSkrbic commented:
Tests for BOOM-847 **OK**

See added screenshots
![z0-screenshot_dsk_2016 07 11_10h14m24s_043_](https://cloud.githubusercontent.com/assets/2835956/16724007/bcf248c4-4750-11e6-8606-d46dc1474f01.jpg)
![z1-screenshot_dsk_2016 07 11_10h12m30s_042_](https://cloud.githubusercontent.com/assets/2835956/16724008/bcf2bf2a-4750-11e6-9c7e-4471330308a5.jpg)

Shudrum commented:
Thank you 👍 

Be careful of the lines length next time please :)

Quetzacoalt91 commented:
Yes sir !

vincentbz commented:
![happy2](https://cloud.githubusercontent.com/assets/7567060/16760986/7ca20f38-481e-11e6-9e5b-a299fac0fe74.gif)
